ethclient/ethclient.go

```diff
@@ -258,6 +258,19 @@ func (ec *Client) SubscribeNewHead(ctx context.Context, ch chan<- *types.Header)
 
 // State Access
 
+// NetworkID returns the network ID (also known as the chain ID) for this chain.
+func (ec *Client) NetworkID(ctx context.Context) (version *big.Int, err error) {
+	version = big.NewInt(0)
+
+	var ver string
+	if err = ec.c.CallContext(ctx, &ver, "net_version"); err != nil {
+		return version, err
+	}
+
+	version.SetString(ver, 10)
+	return
+}
```

Comment:
I looked at the spec a bit too quickly and mistook "id" for "result". The string parsing is fine then, but please check for errors.